source/docs/3/basic_filtering_and_styling_with_expressions.rst

```diff
@@ -0,0 +1,120 @@
+Basic Filtering and Styling with Expressions (QGIS3)
+====================================================
+
+Expressions offer a powerful way to manipulate attribute values, geometries and variables to dynamically modify map features on the fly. This tutorial is an introduction to some common expressions and how they are used in map making. 
+
+.. note::  
+Tutorial contributed by: `Steven Kim <https://geographyclub.github.io/>`_
+    
+Overview of the task
+--------------------
+
+We will use expressions to filter features by attribute from a Natural Earth country layer, and then style them to our liking. We will create a map of South East Asia with each country styled in a shade of green.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/output.png
+    :align: center
+	
+
+Get the data
+------------
+
+1. `Natural Earth <https://www.naturalearthdata.com/downloads/>` has the admin shapefiles for contries. On the downloads page look for :guilabel:`Cultural` option under the ``Medium scale data``. 
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/data1.png
+    :align: center
+	
+2. Download the ``Admin 0 - Countries`` shapefile. You will see the ``ne_50m_admin_0_countries.zip`` file downloaded which is used for this tutorial.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/data2.png
+    :align: center
+
+
+For convenience, you may directly download a copy of the above layer from below:
+`ne_50m_admin_0_countries.zip <http://www.qgistutorials.com/downloads/ne_50m_admin_0_countries.zip>`
```

Comment:
missing _ in url so it is not formatted correctly. Please build once and check to catch such errors